setuptools/build_meta.py

```diff
@@ -369,7 +369,7 @@ def prepare_metadata_for_build_wheel(
         return self._bubble_up_info_directory(metadata_directory, ".dist-info")
 
     def _build_with_temp_dir(
-        self, setup_command, result_extension, result_directory, config_settings
+        self, setup_command, result_extension, result_directory, config_settings, arbitrary_args=False
```

Comment:
The CI error is probably a formatting error in this line:

```suggestion
        self,
        setup_command,
        result_extension,
        result_directory,
        config_settings,
        arbitrary_args=(),
```

(untested suggestion)
(type of `arbitrary_args` changed to match other suggestions)

Reply:
applied as suggested